test/AWSConfig.jl

```diff
@@ -19,4 +19,18 @@
 
         @test config.credentials.access_key_id == access_key_id
     end
+
+    @testset "default profile section names" begin
+        allowed_default_sections = ["default", "profile default"]
+        mktemp() do config_path, _
+            for default_section_str in allowed_default_sections
+                config = """
+                [$default_section_str]
+                region = xx-yy-1
+                """
+                write(config_path, config)
+                @test aws_get_region(; profile="default", config=config_path) == "xx-yy-1"
+            end
+        end
+    end
```

Comment:
I believe this is the proper syntax.

```suggestion
    @testset "$(profile) section names" for profile in ["default", "profile_default"]
        mktemp() do config_path, _
                config = """
                [$(profile)]
                region = xx-yy-1
                """
                write(config_path, config)
                @test aws_get_region(; profile="default", config=config_path) == "xx-yy-1"
            end
        end
    end
```

Reply:
It's nbd, I just find different cases in a `@testset` are easier to debug.